src/fundus/publishers/base_objects.py

```diff
@@ -19,8 +20,17 @@ class PublisherSpec:
     request_header: Dict[str, str] = field(default_factory=dict)
 
 
+class PublisherEnumMeta(EnumMeta):
+    def __str__(self) -> str:
+        representation = f"Publisher Collection {self.__name__} containing {len(self)} publisher(s), including:"
```

Comment:
```suggestion
        representation = f"Region {self.__name__!r} containing {len(self)} publisher(s), including:"
```

Either `Region` or `Country` or nothing, that's technically not a PublicherCollection and I think that could confuse people here.